docs/changelog.rst

```diff
@@ -5,10 +5,14 @@ CHANGELOG
 2.89.1+dev (XXXX-XX-XX)
 -----------------------
  
+**New features**
+
+- Add new command to reorder pathaggregations of topologies
```

Comment:
Can you specify the name of the command, as I am not sure to find its name...
Thanks.

Reply:
I am adding documentation then is it necessary ?

Reply:
OK thanks